src/MediaCollections/Models/Concerns/PathUuidSafe.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Spatie\MediaLibrary\MediaCollections\Models\Concerns;
+
+use Illuminate\Support\Facades\Storage;
+use Spatie\MediaLibrary\MediaCollections\Models\Media;
+use Spatie\MediaLibrary\Support\PathGenerator\PathGeneratorFactory;
+
+trait PathUuidSafe
+{
+    public static function bootPathUuidSafe()
+    {
+        if (! config('media-library.path_uses_uuid', false)) {
```

Comment:
The `false` default can be removed, the config value is always present.

Reply:
Ok